src/utils/date.ts

```diff
@@ -1,5 +1,5 @@
-// TODO: Remove date-fns and covert to use @date-io/date-fns
-// https://github.com/dmtrKovalenko/date-io#projects
+// TODO: Remove date-fns and convert to use the native JS formatter
+// @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat
```

Comment:
It seems like the [`@date-io/date-fns`](https://github.com/dmtrKovalenko/date-io#interface) interface doesn't expose all of the features of `date-fns` that we use. I don't think we can remove it entirely.

`Intl.RelativeTimeFormat` looks like a good replacement for `formatDistanceToNow` (in `formatTimeInWords`) though.